pycodestyle.py

```diff
@@ -1476,10 +1485,15 @@ def _is_eol_token(token, _eol_token=_is_eol_token):
 
 def _get_parameters(function):
     if sys.version_info >= (3, 3):
-        return [parameter.name
-                for parameter
-                in inspect.signature(function).parameters.values()
-                if parameter.kind == parameter.POSITIONAL_OR_KEYWORD]
+        try:
+            return [parameter.name
+                    for parameter
+                    in inspect.signature(function).parameters.values()
+                    if parameter.kind == parameter.POSITIONAL_OR_KEYWORD]
+        except ValueError:
+            """Catches "No signature found for builtin <built-in method
```

Comment:
We merged the other code that doesn't rely on scanning the definitions in the module to find checks. Is this still necessary now? You'll need to rebase so I think this is probably unnecessary at this point.